Dockerfile

```diff
@@ -60,6 +60,7 @@ ENTRYPOINT ["./docker-entrypoint.sh"]
 # STore static files under /var to not conflict with development volume mount
 ENV STATIC_ROOT /var/tunnistamo/static
 ENV NODE_MODULES_ROOT /var/tunnistamo/node_modules
+ENV APP_URL /
```

Comment:
If I'm not mistaken this isn't really a URL, just a path. Something like `/path/to/my/app`. The variable could be named accordingly.

Reply:
It is true that this can only be URL path as opposed to STATIC_URL and MEDIA_URL which can be full URLs. Would APP_URL_PATH be a good name?

Reply:
I suppose that works.